packages/interfaces/src/service/product-analytics.interface.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * The {@link ProductAnalytics} service lets you track client-side behavior and collect
+ * visitor insights.
+ *
+ * @experimental
+ */
+export interface ProductAnalytics {
+    /**
+     * Track events
+     *
+     * @param {string} eventName name of the event.
+     * @param {Record<string, unknown>} config an object containing a key-value
+     * pairs to be associated with an event.
+     */
+    track(eventName: string, config: Record<string, unknown>): void;
```

Comment:
Why do we have `unknown` here and not `any`? I'm not sure I know the difference. But if you have any then the user can input anything? 🤔 
```suggestion
    track(eventName: string, config: Record<string, any>): void;
```

Reply:
As I understand the `unknown` type is the type-safe counterpart of `any`. Anything is assignable to `unknown`, but `unknown` isn't assignable to anything.
For example

```
let age: number;
let userAge: unknown;

userAge = 'This is some age';
userAge = 20;

age = userAge;
````

The statement age=userAge gives an error. I would say unknown is a better choice. @jgroth

Reply:
`unknown` is safer to use than `any` because it enforces type checking
I found this article that probably explains it better than I do :slightly_smiling_face: 
https://dmitripavlutin.com/typescript-unknown-vs-any/#1-unknown-vs-any

Reply:
What happened to `string | number | boolean` that were there before? :slightly_smiling_face:

Reply:
Thanks for the explanation, let's keep it as unknown 🙂